redis-hbo-provider/src/main/java/com/facebook/presto/statistic/RedisProviderPlugin.java

```diff
@@ -50,8 +50,8 @@ public RedisProviderPlugin(Map<String, String> configs)
     public Iterable<HistoryBasedPlanStatisticsProvider> getHistoryBasedPlanStatisticsProviders()
     {
         if (this.propertyMap == null) {
-            log.info("Redis Provider Plugin returned an empty instance");
-            return ImmutableList.of(EmptyPlanStatisticsProvider.getInstance());
+            log.error("Redis Provider Plugin returned an empty instance");
```

Comment:
When is `propertyMap` set to null--is this just the default value?  How is it able to reach this path?

Reply:
Don't think it is ever set to null, can remove too but good to have?